php/sw-fw-less/config.yaml

```diff
@@ -0,0 +1,16 @@
+framework:
+  github: luoxiaojun1992/sw-fw-less
+  version: 1.0
```

Comment:
last : should be **preview**

I'm watching for you release, so I'll update when getting to `1.0`

Reply:
Yes. I have changed the version to **preview**.

Reply:
:+1: